### PR TITLE
chore: migrate synth.py to bazel

### PR DESCRIPTION
**Pull request by vam-google**
The changes in grpc stubs are caused by the gRPC upgrade from 1.10 (more than a year old) to 1.27 (same version which is used as runtime dependency)

Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for context and/or discussion)

### CODE REVIEW

**Comment by chingor13** [comment]
Did you mean 1.27 for the gRPC version?

File: grpc-google-cloud-language-v1/clirr-ignored-differences.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <!-- TODO: remove after 1.99.3 released -->
+    <differenceType>6001</differenceType>
+    <className>com/google/cloud/language/v1/*Grpc</className>
+    <field>METHOD_*</field>
```
Comment (elharo) [inline grpc-google-cloud-language-v1/clirr-ignored-differences.xml]: can this be more specific?

File: synth.metadata [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
...e_transform\n\nPiperOrigin-RevId: 297405063\n\n0aba1900ffef672ec5f0da677cf590ee5686e13b\ncluster: use square brace for cross-reference\n\nPiperOrigin-RevId: 297204568\n\n5dac2da18f6325cbaed54603c43f0667ecd50247\nRestore retry params in gapic config because securitycenter has non-standard default retry params.\nRestore a few retry codes for some idempotent methods.\n\nPiperOrigin-RevId: 297196720\n\n1eb61455530252bba8b2c8d4bc9832960e5a56f6\npubsub: v1 replace IAM HTTP rules\n\nPiperOrigin-RevId: 297188590\n\n80b2d25f8d43d9d47024ff06ead7f7166548a7ba\nDialogflow weekly v2/v2beta1 library update:\n  - updates to mega agent api\n  - adding field mask override control for output audio config\nImportant updates are also posted at:\nhttps://cloud.google.com/dialogflow/docs/release-notes\n\nPiperOrigin-RevId: 297187629\n\n0b1876b35e98f560f9c9ca9797955f020238a092\nUse an older version of protoc-docs-plugin that is compatible with the specified gapic-generator and protobuf versions.\n\nprotoc-docs-plugin >=0.4.0 (see commit https://github.com/googleapis/protoc-docs-plugin/commit/979f03ede6678c487337f3d7e88bae58df5207af) is incompatible with protobuf 3.9.1.\n\nPiperOrigin-RevId: 296986742\n\n"
```
Comment (elharo) [inline synth.metadata]: Is all this needed? Some of it seems like it might have slipped in from other products?

Comment (chingor13) [reply synth.metadata]: Yes, we use the metadata file and the internalRef is intentional. It's included in every commit in googleapis/googleapis

File: grpc-google-cloud-language-v1/clirr-ignored-differences.xml (hunk 1 above)
Comment (chingor13) [reply grpc-google-cloud-language-v1/clirr-ignored-differences.xml]: We could explicitly list out every removed field, but it seems unnecessary extra work. These files are complete generated and we know this only affects fields prefixed by `METHOD_`

**Comment by vam-google** [comment]
@chingor13 Yes, I made a typo, it should be 1.27.

**Pull request merged**